src/packages/EmbedIO/Net/Internal/SystemHttpResponse.cs

```diff
@@ -1,6 +1,8 @@
 ﻿using System;
+using System.Collections;
```

Comment:
Are you sure these two added usings are necessary? You made no other changes to this file.

Reply:
Solved in later commit. 👍